extensions/wikia/RobotsTxt/classes/WikiaRobots.class.php

```diff
@@ -82,9 +82,6 @@ class WikiaRobots {
 		'/fandom?p=',
 		//Mobile Wiki search URL
 		'/search',
-
-		// AdEngine recovery api
-		'/wikia.php?controller=AdEngine2ApiController&method=getBTCode'
```

Comment:
Should we add here the other rec endpoints from AE3?

Reply:
Yep, added